Equilobe.DailyReport.SL/ReportService.cs

```diff
@@ -18,8 +21,33 @@ public class ReportService : IReportService
         public ITaskSchedulerService TaskSchedulerService { get; set; }
 
         public JiraRequestContext JiraRequestContext { get; set; }
+        public TimeSpan OffsetFromUtc { get; set; }
```

Comment:
Same for the offset. This should be a parameter on the service methods, not a property on the service